testing/rabbitmq/rabbitmq.initd

```diff
@@ -0,0 +1,38 @@
+#!/sbin/openrc-run
+
+pidfile=/var/run/rabbitmq/rabbitmq.pid
+name=rabbitmq
+daemon=/usr/sbin/rabbitmq-server
+
+depend() {
+	need net localmount
+	after firewall
+}
+
+start() {
+	ebegin "Starting ${name}"
+	# We need to test if /var/run/rabbitmq exists, since scirpt will fail if it does not
+	[ ! -e /var/run/rabbitmq ] && \
+		mkdir /var/run/rabbitmq && \
+		chown rabbitmq:rabbitmq /var/run/rabbitmq
```

Comment:
OpenRC provides a handy helper function for this, `checkpath` (see [man openrc-run](http://manpages.org/openrc-run/8)).

```diff
-	# We need to test if /var/run/rabbitmq exists, since scirpt will fail if it does not
-	[ ! -e /var/run/rabbitmq ] && \
-		mkdir /var/run/rabbitmq && \
-		chown rabbitmq:rabbitmq /var/run/rabbitmq
+	checkpath -m 700 -o rabbitmq:rabbitmq -d "$(dirname "$pidfile")"
```